pkg/util/paramtable/service_param.go

```diff
@@ -353,7 +362,27 @@ func (p *MetaDBConfig) Init(base *BaseTable) {
 		Export:       true,
 	}
 	p.MaxIdleConns.Init(base.mgr)
+}
+
+// /////////////////////////////////////////////////////////////////////////////
+// --- mq ---
+
+// MQConfig represents the configuration settings for the message queue.
+type MQConfig struct {
+	Type ParamItem `refreshable:"false"`
+}
 
+// Init initializes the MQConfig object with a BaseTable.
+func (p *MQConfig) Init(base *BaseTable) {
+	p.Type = ParamItem{
+		Key:          "mq.type",
+		Version:      "2.2.0",
```

Comment:
should be 2.3.0?

Reply:
new commit changed:
all field of this commit in yaml is changed to 2.3.0